content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.endswith()'
+Description: 'Checks whether or not a string ends with the given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+TThe `.endswith()` method returns `True` if a particular value is attached to the end of a given string. Otherwise, it returns `False`.
+
+## Syntax
+
+Given a  `string`, the `.endswith()` method can be used in the following way:
+
```

Comment:
Was there a code snippet that was supposed to go here (using \```pseudo)? Something like this?

```pseudo
string.endswith(value, start, end)
```